src/value/createValueMockFactory.ts

```diff
@@ -252,4 +253,11 @@ export class ValueMockFactory {
 	>(config?: WithoutFakerConfig<MockRichTextValueConfig<Model>>) {
 		return richText({ ...config, faker: this.faker });
 	}
+
+	table<
+		Model extends
+			prismic.CustomTypeModelTableField = prismic.CustomTypeModelTableField,
+	>(config?: WithoutFakerConfig<MockTableValueConfig<Model>>) {
+		return table({ ...config, faker: this.faker });
+	}
 }
```

Comment:
These are also sorted alphabetically so I would put this (and ricText) in the correct place(s) further up.

Reply:
I didn't notice, I applied this change because this is exposed in the API, but I'm not sure if alphabetically is the best way to organize this.